lib/foreman_inventory_upload/async/upload_report_job.rb

```diff
@@ -8,6 +8,14 @@ def self.output_label(label)
       end
 
       def perform(filename, organization_id)
+        label = UploadReportJob.output_label(organization_id)
+        if Setting[:content_disconnected]
+          progress_output_for(label) do |progress_output|
+            progress_output.write_line('Upload was stopped since this disconnected setting is enabled')
```

Comment:
```suggestion
            progress_output.write_line('Upload was stopped since disconnected mode setting is enabled for content on this instance.')
```

Reply:
Fixed, thanks it sounds better :+1: